resources/lib/modules/bluetooth.py

```diff
@@ -34,10 +36,12 @@ class bluetooth(modules.Module):
     @log.log_function()
     def __init__(self, oeMain):
         super().__init__()
+        self.oe = oeMain
```

Comment:
I haven’t retested. But guessing (does this fix) #5398

Reply:
No, but it should fix a potential `AttributeError` at bluetooth.py:186 I still don't know that caused that error.

Reply:
A quick shot:

The code is using oe.winOeMain but the global variable may be deleted in oe.py.

Removing the two lines at https://github.com/LibreELEC/service.libreelec.settings/blob/9e62eaae3083cb30e1efc45a9c21ed1918283275/resources/lib/oe.py#L626-L627 let the error disappear for me.

It was added eight years ago with https://github.com/LibreELEC/service.libreelec.settings/commit/13c32b6385bc55be0971e0eb1d6417adf2a7b308

Reply:
I know the motivation behind this code, however, IMO this implementation is not quite correct. But your comment has made me think that probably a BT discovery thread outlives the main addon thread after the window was closed and still tries to update already GC-ed window. At least I see that the thread is stopped without calling .join() that is not correct because you have no guarantee that the thread has actually stopped.